### PR TITLE
fixed homepage side navbar

### PR DESCRIPTION
**Pull request by Grub2801**
homepage side nav bar works
The playspace_app folder (that we deleted monday morning) has reappeared in the master branch and subsequently, it is in this branch.

### CODE REVIEW

**Comment by f3r** [comment]
> This branch has conflicts that must be resolved

I merged Ada's pull request to master, and now your branch can't merge with master directly. To solve the conflicts, you pull the new master to your local, then from this branch (homepage-bug), you do `git rebase master`, fix the errors, `git add` the modified files and then, instead of committing, you do `git rebase --continue`

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -12,72 +12,66 @@
...
-      <!-- Sidebar -->
-      <div id="sidebar-wrapper">
-          <ul class="sidebar-nav">
+    <div id="wrapper">
 
-            <li class="sidebar-brand">PlaySpace</li>
-            <li><a href="#">Find a PlaySpace</a></li>
-            <li><%= link_to 'About', about_path %></li>
+        <!-- Sidebar -->
+        <div id="sidebar-wrapper">
```
Comment (dshcheung) [inline app/views/layouts/application.html.erb]: please check your indentations.

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -12,72 +12,66 @@
...
+    <script>
+    $("#menu-toggle").click(function(e) {
+        e.preventDefault();
+        $("#wrapper").toggleClass("toggled");
+    });
+    </script>
 
 </body>
-</html>
+</html>
```
Comment (dshcheung) [inline app/views/layouts/application.html.erb]: add a new line here

File: config/routes.rb [hunk 3]
```diff
@@ -14,4 +14,4 @@
   resources :categories, only: [:index, :show, :new, :create] do
     resources :playspaces, only: [:index, :show, :new, :create]
   end
-end
+end
```
Comment (dshcheung) [inline config/routes.rb]: add a new line here

**Comment by Grub2801** [comment]
had two files with errors.
changed one, forgot to change the second before I ran git rebase --continue
went back to change the file with errors
can I git add. and commit and push it back to origin?

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -10,74 +10,57 @@
...
 
-      <!-- Sidebar -->
-      <div id="sidebar-wrapper">
+    <div id="wrapper">
+        <!-- Sidebar -->
+        <div id="sidebar-wrapper">
           <ul class="sidebar-nav">
-
             <li class="sidebar-brand">PlaySpace</li>
             <li><a href="#">Find a PlaySpace</a></li>
```
Comment (Grub2801) [inline app/views/layouts/application.html.erb]: change indentations

**Pull request merged**